homeassistant/components/flux_led/__init__.py

```diff
@@ -1 +1,118 @@
-"""The flux_led component."""
+"""The Flux LED/MagicLight integration."""
+import asyncio
+import copy
+from datetime import timedelta
+import logging
+
+from flux_led import BulbScanner
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_DEVICES, CONF_HOST, CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.event import async_track_time_interval
+from homeassistant.util import Throttle
+
+from .const import (
+    CONF_AUTOMATIC_ADD,
+    DEFAULT_NETWORK_SCAN_INTERVAL,
+    DOMAIN,
+    SIGNAL_ADD_DEVICE,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["light"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Flux LED/MagicLight component."""
+    hass.data.setdefault(DOMAIN, {})
```

Comment:
A future clean up is to move this to `async_setup_entry` and remove `async_setup`. The latter isn't needed anymore.